README.md

```diff
@@ -124,12 +124,39 @@ request.applePayDisabled = !canMakePayments
 
 ### 3D-Secure + Drop-In
 
-The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, then just enable it in the BTDropInRequest and set an amount.
+The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, then just enable it in the BTDropInRequest and set an amount. Then, create a BTThreeDSecureRequest() object and set as many fields on it as possible; the more fields that are set, the less likely it is for a user to be presented with a challenge. Make sure to attach this object to the BTDropInRequest before use.
 
 ```swift
-let request =  BTDropInRequest()
+let request = BTDropInRequest()
 request.threeDSecureVerification = true
 request.amount = "1.00"
+
+let threeDSecureRequest = BTThreeDSecureRequest()
+threeDSecureRequest.threeDSecureRequestDelegate = self
+
+threeDSecureRequest.amount = 1.0
+threeDSecureRequest.email = "test@example.com"
+threeDSecureRequest.requested = .version2
```

Comment:
just doublechecking because new to iOS -  there should be a `.` in `.version2`

Reply:
Yep, that's Swift shorthand for the enum value `BTThreeDSecureVersion2`.